### PR TITLE
nu: add page

### PR DESCRIPTION
**Pull request by nicokosi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Note: I took [bash's page](https://github.com/tldr-pages/tldr/blob/master/pages/common/bash.md) as a model (but used long options).

### CODE REVIEW

**Comment by nicokosi** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/776709015) for this PR failed with the following error(s):
> 
> ```
> pages/common/nu.md:
> Error: Parse error on line 4:
> ...ps://www.nushell.sh>- Start an interac
> -----------------------^
> Expecting 'END_INFORMATION_LINK_URL', got 'NEWLINE'
> pages/common/nu.md:4: TLDR011 Page never contains more than a single empty line
> ```
> 
> Please fix the error(s) and push again.

Hey 🤖, it's fixed. 😉

File: pages/common/nu.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# nu
+
+> Nushell, a "new type of shell": takes a modern, structured approach to your commandline.
```
Comment (marchersimon) [inline pages/common/nu.md:3]:
```suggestion
> Nushell - a modern, structured approach to your commandline.
```
This sounds a bit more tldr-conform, what do you think?

Comment (nicokosi) [reply pages/common/nu.md:3]: I prefer my take that mixes the official moto (`a "new type of shell"`, that explain the name) with my own explanation (`structured approach to your commandline`). 😇

**Review by marchersimon — APPROVED** [review]

File: pages/common/nu.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/nu.md:3]: Yeah, it does, but I'm not sure about the hypen used there 🤔
Perhaps this?
```suggestion
> Nushell (new shell) takes a modern, structured approach to your command-line.
```

Comment (nicokosi) [inline pages/common/nu.md:3]:
```suggestion
> Nushell ("a new type of shell") takes a modern, structured approach to your command-line.
```

Comment (nicokosi) [reply pages/common/nu.md]: OK 👍. I did a mix of your comments and mine in commit e7b66f4.

File: pages/common/nu.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> Nushell ("a new type of shell") takes a modern, structured approach to your command-line.
+> More information: <https://www.nushell.sh>.
+
+- Start an interactive shell session:
+
+`nu`
+
+- Execute a command and then exit:
+
+`nu --commands "{{command}}"`
```
Comment (navarroaxel) [inline pages/common/nu.md:12]: How can I send arguments to the target command? (check [nix-shell](https://github.com/tldr-pages/tldr/blob/master/pages/common/nix-shell.md) and [kde-inhibit](https://github.com/tldr-pages/tldr/blob/master/pages/linux/kde-inhibit.md) for examples).

Comment (nicokosi) [reply pages/common/nu.md:12]: Not sure it applies to `nushell`. If I understand well (I am a `nushell` newbie 😇), `--commands`'s value is a function pipeline (function(s) with arguments or/and operators):  
```
nu --commands "echo 42"
```
```
nu --commands "ls | sort-by size | reverse"
```

Comment (navarroaxel) [reply pages/common/nu.md:12]: Cool, so it's like `nix-shell`. We can add this to make it clear in the example:
```suggestion
`nu --commands "{{command}} {{command_arguments}}"`
```

Comment (nicokosi) [reply pages/common/nu.md:12]: Hmm, I think we cannot say that a Nushell command has optional arguments. If I understand well, commands can have parameters and can be combined with operators, but I am not sure we have to mention this in `tldr`. :nerd_face:

Comment (sbrl) [reply pages/common/nu.md:12]: Yeah, I think it's best to keep things simple here. Let's keep this as-is.

**Review by Waples — APPROVED** [review]

**Pull request merged**